corecito.py

```diff
@@ -2,118 +2,166 @@
 import asyncio
 import time
 import logging
+import yaml
+import sys
+from os.path import exists
 import cryptocom.exchange as cro
 from cryptocom.exchange.structs import Pair
 from cryptocom.exchange.structs import PrivateTrade
 
-CRYPTOCOM_API_KEY = "MY_KEY"
-CRYPTOCOM_API_SECRET = "MY_SECRET"
-
-DEFAULT_CORE_NUMBER = 2 # Ethers
-MAX_DECIMALS_BUY = 6 # This is the limit Crypto.com has on ETH_BTC pair https://crypto.com/exchange/trade/spot/ETH_BTC
-MAX_DECIMALS_SELL = 3 # This is the limit Crypto.com has on ETH_BTC pair https://crypto.com/exchange/trade/spot/ETH_BTC
+# CRYPTOCOM_API_KEY = "MY_KEY"
+# CRYPTOCOM_API_SECRET = "MY_SECRET"
+#
+# DEFAULT_CORE_NUMBER = 2 # Ethers
+# config['max_decimals_buy'] = 6 # This is the limit Crypto.com has on ETH_BTC pair https://crypto.com/exchange/trade/spot/ETH_BTC
+# config['max_decimals_sell'] = 3 # This is the limit Crypto.com has on ETH_BTC pair https://crypto.com/exchange/trade/spot/ETH_BTC
```

Comment:
Your improvements made this code obsolete so we don't need it anymore. Would you like to remove it?